util.c

```diff
@@ -5599,32 +5600,67 @@ Perl_xs_handshake(const U32 key, void * v_my_perl, const char * file, ...)
         if (PERLDB_LINE_OR_SAVESRC)
             (void)gv_fetchfile(file); */
     }
-
+    die_mismatched_rmv_c_args:
```

Comment:
What does `rmv` stand for?

Reply:
Ah. I wouldn't have guessed that. I would suggest either spelling it out in the label (`die_mismatched_remove_c_args`) or shortening it to just `rm` (because I figure most of us are familiar with the unix command).

Reply:
I think rev #2/#3 is good enough to commit regarding this line of code. Unless more people disagree, I'd rather keep "rmv" vs "remove" in the goto label. Not a best practice for code to use full sentences as identifiers or tokens, substituting all white space with alot of "_"s.

I don't have much opinion on "rmv" vs "remove" in this case. Im just follow generic best practice of dont write full sentences as identifiers I was taught long ago.